db/migrate/20160729094457_add_columns_to_ansible_role.rb

```diff
@@ -0,0 +1,12 @@
+#:nodoc:
```

Comment:
The rubocop doc requirement is there to explain the 'why's of this migration - the 'how' is self-explanatory